openpype/settings/ayon_settings.py

```diff
@@ -748,7 +748,19 @@ def _convert_nuke_project_settings(ayon_settings, output):
     )
 
     new_review_data_outputs = {}
-    for item in ayon_publish["ExtractReviewDataMov"]["outputs"]:
+    outputs_settings = None
+    # just in case that the users having old presets in outputs setting
+    deprecrated_review_settings = ayon_publish["ExtractReviewDataMov"]
+    current_review_settings = (
+        ayon_publish["ExtractReviewDataBakingStreams"]
+    )
+    if not deprecrated_review_settings["enabled"]:
+        if current_review_settings["enabled"]:
+            outputs_settings = current_review_settings["outputs"]
+    else:
+        outputs_settings = deprecrated_review_settings["outputs"]
```

Comment:
I suspect this might also be more readable with the logic changed like [here](https://github.com/ynput/OpenPype/pull/5623/files#r1338759890).